http/obmc_shell.hpp

```diff
@@ -58,45 +58,19 @@ class Handler : public std::enable_shared_from_this<Handler>
         {
             // CHILD
 
-            // sets the effective user ID
-            // as service user
-            bool isUidSet = false;
             if (auto userName = session->getUserName(); !userName.empty())
             {
-                if (struct passwd* pw = getpwnam(userName.c_str());
-                    pw != nullptr)
-                {
-                    int uidr = setuid(pw->pw_uid);
-                    if (!uidr)
-                    {
-                        isUidSet = true;
-                    }
-                    else
-                    {
-                        BMCWEB_LOG_ERROR << "sets service user failed Error: "
-                                         << uidr;
-                    }
-                }
-                else
-                {
-                    BMCWEB_LOG_ERROR << "getpwnam return null passwd";
-                }
+                execl("/bin/login", "/bin/login", "-f", userName.c_str(), NULL);
```

Comment:
do we want to cast the last argument to (char*) NULL? 

From the execl man page:

The list of arguments must be terminated by a NULL pointer, and, since these are variadic functions, this pointer must be cast (char *) NULL.

Reply:
I don't this we need to do that

Reply:
FYI: https://linuxhint.com/exec_linux_system_call_c/